Makefile

```diff
@@ -1,18 +1,26 @@
-TOOL_BIN = bin/gotools/$(shell uname -s)-$(shell uname -m)
+TOOL_BIN := bin/gotools/$(shell uname -s)-$(shell uname -m)
+PROTO_FILES := $(shell find proto/ -type f -name '*.proto')
 
-PATH_WITH_TOOLS="`pwd`/$(TOOL_BIN):`pwd`/node_modules/.bin:${PATH}"
```

Comment:
Makefiles do not support backticks. This works sort-of by accident: the backticks are inserted verbatim into `bash`, which then expands them.

This is brittle, so I replaced `pwd` with `$(shell pwd)`, which Makefile will properly expand itself. I also switch the `=` (evaluate every use) with `=:` (evaluate at assignment)